src/main/java/ch/njol/skript/conditions/CondIsDivisibleBy.java

```diff
@@ -0,0 +1,64 @@
+package ch.njol.skript.conditions;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Condition;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Checker;
+import ch.njol.util.Kleenean;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Divisible By")
+@Description("Check if a number is divisible by another number.")
+@Examples({
+	"if 5 is divisible by 5:",
+	"if 11 cannot be divided by 10:",
+})
+@Since("INSERT VERSION")
+public class CondIsDivisibleBy extends Condition {
+
+	static {
+		Skript.registerCondition(CondIsDivisibleBy.class,
+			"%numbers% (is|are) divisible by %number%",
+			"%numbers% (isn't|is not|aren't|are not) divisible by %number%",
+			"%numbers% can be evenly divided by %number%",
+			"%numbers% (can't|cannot|can not) be [evenly] divided by %number%");
```

Comment:
I feel like the use of `evenly` should be consistent throughout all of the syntax. I think it should be required for all syntaxes but if you want to make it optional that would be okay too.

Reply:
required in all please :D